src/lib.rs

```diff
@@ -76,14 +76,14 @@ pub struct GitHubClaims {
     pub iat: u64,
 }
 
+/// Default URL for fetching GitHub OIDC tokens
+pub const DEFAULT_GITHUB_OIDC_URL: &str = "https://token.actions.githubusercontent.com";
```

Comment:
😍 
Yay!

Took me a few minutes to find this URL when I needed it 😆 Was hard to find in Github Docs so I resorted to giving in your oidc service repo for it! Much better to have it here!